README.md

```diff
@@ -216,37 +215,6 @@ cargo run --release --bin pathfinder -- --help
 docker run --rm eqlabs/pathfinder
 ```
 
-The configuration file uses the `toml` format:
-
-```toml
-# The address we will host the RPC API at. Defaults to "127.0.0.1:9545"
-http-rpc = "127.0.0.1:1235"
-# The directory the node will use to store its data. Defaults to the current directory.
-data-directory = "..."
-# Override the Sequencer gateway address with your own. This is can be useful if you
-# have a caching proxy in front of the actual Sequencer gateway. If you're unsure
-# of what this does, then you don't need it.
-sequencer-url = "https://..."
-# Set the number of Python subprocesses pathfinder starts. These processes are used
-# to service the `starknet_call` JSON-RPC method and their number limits the maximal
-# number of call requests that can be processed in parallel. Defaults to 2.
-python-subprocesses = 2
-# Whether to enable SQLite write-ahead logging. Defaults to true.
-sqlite-wal = true
-# Whether to enable pending support.
-poll-pending = true
-# The address to host the monitoring API at. Defaults to disabled.
-monitor-address = "127.0.0.1:54321"
-# Use Goerli Testnet 2 instead of Goerli Testnet. Defaults to false.
-testnet2 = true
-
-[ethereum]
-# This is required and must be an HTTP(s) URL pointing to your Ethereum node's endpoint.
-url      = "https://goerli.infura.io/v3/..." #
-# The optional password for your Ethereum endpoint.
-password = "..."
-```
-
```

Comment:
Hmm, do we want to remove this part? I thought this is something we'd leave for a future release...

Reply:
I figured if its deprecated we shouldn't advertise it

Reply:
OK.